lasagne/tests/test_utils.py

```diff
@@ -37,7 +37,7 @@ def test_collect_shared_vars():
 def test_one_hot():
     from lasagne.utils import one_hot
     a = np.random.randint(0, 10, 20)
-    b = np.zeros((a.size, a.max()+1))
+    b = np.zeros((a.size, a.max() + 1))
```

Comment:
Same here, please do not introduce whitespace changes outside of what you're working on. This makes it more difficult to track changes later.